tdr-snapshot-examples/get_snapshot.py

```diff
@@ -0,0 +1,37 @@
+# Gets snapshot information based off a provided snapshot id
+
+import json
+
+import click
+from obtain_token import obtain_session
+
+
+tdr_domain = "data.terra.bio"
+payload = {
+    "offset": 0,
+    "limit": 30,
+    "sort": "datarepo_row_id",
+    "direction": "asc",
+    "filter": "",
+}
+headers = {
+    "User-Agent": "GPO Bound Query",
+    "Content-Type": "application/json",
+}
+
+
+def get_snapshot(session):
+    snapshot_id = click.prompt("Enter snapshot ID")
+    content_response = session.post(
+        url=f"https://{tdr_domain}/api/repository/v1/snapshots/{snapshot_id}/data/sample",
+        headers=headers,
+        data=json.dumps(payload),
+    )
+    content_response.raise_for_status()
+    return content_response.json()
```

Comment:
This may work better for clients if we pass in the snapshot ID

Use the click in the Main and pass that in.

Reply:
Done by you 👍